### DISCUSSION TITLE
Tiling compatibility ?

### DISCUSSION THREAD

**Discussion by EdiDD** [ddterm/gnome-shell-extension-ddterm#98] [Q&A]
Hi, is it possible to add tiling compatibility like in tilix terminal emulator [https://gnunn1.github.io/tilix-web/](url)  ?
It think it will be best terminal emulator for gnome on wayland then.

**Comment by amezin** [comment]
https://github.com/amezin/gnome-shell-extension-ddterm/issues/42
